### ISSUE TITLE
Installing and updating netdata to stable releases only

### ISSUE THREAD

**Issue by elfgoh** [firehol/netdata#2479]
I would like more control over the version of netdata installed. Currently, it seems like the installation one-liner installs the latest commit.

Is it sensible for me to desire to install and update to stable versions only, instead of the bleeding edge?

**Comment by ktsaou** [comment]
You are right.
This however should be handled by the distros. The whole idea is to have them use the latest release, so we should only provide a method for using the latest unreleased.

Unfortunately, 15 months after the first official release of netdata, most mainstream distros have not yet added netdata... I still want to believe we are close to (or at least: closer than ever).

**Comment by Ferroin** [comment]
FWIW, while using a stable release is generally preferable, the upstream Git version is remarkably stable in my experience, and I actually run that on most of my systems with no issues (I've only used it for a couple of months, but I update netdata about twice a week, and I've not had any issues with it failing after an update).

**Comment by ktsaou** [comment]
@Ferroin thanks! I try very hard to keep it like that. Generally, I hate bugs. Especially stability bugs take the first priority on top of everything else and they are fixed asap. I think this is reflected at the open issues list.

**Comment by elfgoh** [comment]
@ktsaou Hmm is it possible to pass the stable release version number as an argument to the build command? Implementation wise I would imagine that the git repo will be checked out to the release branch

**Comment by ktsaou** [comment]
this seems feasible...

The following checks out netdata v1.7 (the released one):

```sh
NETDATA_VERSION="1.7"
git clone -b "v${NETDATA_VERSION}.0" --single-branch --depth 1 https://github.com/firehol/netdata.git netdata-v${NETDATA_VERSION}.0.git
```

It will create directory `netdata-v1.7.0.git`. So, to install it, enter this directory and execute `./netdata-installer.sh`.

**Comment by elfgoh** [comment]
Sounds simple. Would you consider a PR to implement an optional argument that can take in the stable release version?

https://github.com/firehol/netdata/blob/master/kickstart.sh#L330

Specifically, I run `bash <(curl -Ss https://my-netdata.io/kickstart.sh) all --dont-wait`

**Comment by ktsaou** [comment]
Of course, but please don't change the existing default.

If you do it, it would be nice to allow the user to select version with `--version v1.7.0`. Instead of `v1.7.0` the user could use `latest` (the current functionality) or `release` to get the latest released version.

It is generally better though to use the latest git version. For example, yesterday we fixed a nasty bug with #2499

**Comment by ktsaou** [comment]
cleaning up.

**Comment by DavidS3141** [comment]
I was just about to open a new feature request when I found this (through the GitHub beta of related issues, its working amazingly well). So I see from above that it was never the intention to add this functionality to the install script, but some pointed out that it seems not to difficult and after more than a year there is still no official package releases (or am I missing something). I would actually try do it myself, if someone would state that the below proposal seems sensible.

##### Feature idea summary
Currently the `kickstart.sh` installs the current repo state with all the latest bugs. You have released versions. Would it be possible to add a version flag to this installer script for those who need the hopefully more reliable released versions? I know that I could to it by hand, but this seems like a very common feature for other projects which netdata is missing. Ideally the cron updater script would have also this flag and only update if there is a new release, otherwise one would have to update manually.

##### Expected behavior
Default:
```bash
bash <(curl -Ss https://my-netdata.io/kickstart.sh)
```
or
```bash
bash <(curl -Ss https://my-netdata.io/kickstart.sh) --version latest
```
Install the released version 1.11.1 without auto-updates:
```bash
bash <(curl -Ss https://my-netdata.io/kickstart.sh) --no-updates --version 1.11.1
```
Install the latest released version (keyword) w/o auto-updates:
```bash
bash <(curl -Ss https://my-netdata.io/kickstart.sh) --no-updates --version release
```
Install the latest release with auto-updates turned on, but because the version flag is given, the behavior of the auto-updates should be adapted such that only released versions are updated (otherwise it would immediately overwrite):
```bash
bash <(curl -Ss https://my-netdata.io/kickstart.sh) --version release
```
